targets/csharp/source/source/Uunit/PluginManagerTest.cs

```diff
@@ -0,0 +1,125 @@
+#if !DISABLE_PLAYFABCLIENT_API
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace PlayFab.UUnit
+{
+    public class PluginManagerTest : UUnitTestCase
+    {
+        private class CustomSerializerPlugin : ISerializerPlugin
+        {
+            public T DeserializeObject<T>(string serialized)
+            {
+                throw new NotImplementedException();
+            }
+
+            public T DeserializeObject<T>(string serialized, object serializerStrategy)
+            {
+                throw new NotImplementedException();
+            }
+
+            public object DeserializeObject(string serialized)
+            {
+                throw new NotImplementedException();
+            }
+
+            public string SerializeObject(object obj)
+            {
+                throw new NotImplementedException();
+            }
+
+            public string SerializeObject(object obj, object serializerStrategy)
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        private class CustomTransportPlugin : ITransportPlugin
+        {
+            public string Name;
+
+            public async Task<object> DoPost(string urlPath, object request, Dictionary<string, string> headers)
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        /// <summary>
+        /// CLIENT AND SERVER API
```

Comment:
CLIENT AND SERVER API doesn't apply to your tests.
the whole line is irrelevant since you're not testing APIs

Reply:
with a bunch of "ditto's" all through this test file.